Core.lua

```diff
@@ -811,7 +813,7 @@ local function BuildGroupieWindow()
 
     local AllTabButton = CreateFrame("Button", "GroupieTab9", GroupieFrame, "CharacterFrameTabButtonTemplate")
     AllTabButton:SetPoint("LEFT", "GroupieTab8", "RIGHT", -16, 0)
-    AllTabButton:SetText("All")
+    AllTabButton:SetText(L["All"])
```

Comment:
```suggestion
    AllTabButton:SetText(_G.ALL)
```

https://github.com/tekkub/wow-globalstrings/blob/master/GlobalStrings/enUS.lua#L581